--- /home/users/hunter/backup/Makefile~	Thu Sep 26 01:28:37 2002
+++ ../BUILD/ttf2pt1-3.4.2/Makefile	Thu Sep 26 01:30:59 2002
@@ -238,9 +238,9 @@
 	scripts/inst_dir $(MANDIR)/man1 $(OWNER) $(GROUP) 0755
 	scripts/inst_dir $(MANDIR)/man5 $(OWNER) $(GROUP) 0755
 	cp -R $(TXTFILES) $(SUBDIRS) $(SHAREDIR)
-	chown -R $(OWNER) $(SHAREDIR)
-	chgrp -R $(GROUP) $(SHAREDIR)
-	chmod -R go-w $(SHAREDIR)
+	#chown -R $(OWNER) $(SHAREDIR)
+	#chgrp -R $(GROUP) $(SHAREDIR)
+	#chmod -R go-w $(SHAREDIR)
 	scripts/inst_file ttf2pt1 $(BINDIR)/ttf2pt1 $(OWNER) $(GROUP) 0755
 	[ -f $(BINDIR)/t1asm ] || scripts/inst_file t1asm $(LIBXDIR)/t1asm $(OWNER) $(GROUP) 0755
 	sed 's|^TTF2PT1_BINDIR=$$|TTF2PT1_BINDIR=$(BINDIR)|;\
@@ -254,9 +254,6 @@
 		sed 's|TTF2PT1_BINDIR|$(BINDIR)|;\
 			s|TTF2PT1_LIBXDIR|$(LIBXDIR)|;\
 			s|TTF2PT1_SHAREDIR|$(SHAREDIR)|;' <$$i >$(MANDIR)/man1/$$i \
-		&& chown $(OWNER) $(MANDIR)/man1/$$i \
-		&& chgrp $(GROUP) $(MANDIR)/man1/$$i \
-		&& chmod 0644 $(MANDIR)/man1/$$i \
 		|| exit 1; \
 	} done
 
